src/App.js

```diff
@@ -1,16 +1,72 @@
-import React from 'react';
-import styles from './App.module.css'
-import logo from './logo.svg';
-import Form from './Form.js'
+import React, { Component } from "react";
+import "./App.css";
+import Header from "./components/Header.js";
+import Todos from "./components/Todos.js"
+import AddTodo from "./components/AddTodo.js"
 
+export default class App extends Component {
+  state = {
+    todos: [
+      {
+        id: 1,
+        title: "Untold story",
+        completed: false
+      },
 
-function App() {
-  return (
-    <div className={styles.base}>
-      <Form />
+      {
+        id: 2,
+        title: "Take out the trash",
+        completed: false
+      },
 
-    </div>
-  );
-}
+      {
+        id: 3,
+        title: "Real life",
+        completed: false
+      }
+    ]
+  };
 
-export default App;
+  markComplete = (id) => {
+    this.setState({
+      todos: this.state.todos.map(todo => {
+        if (todo.id === id) {
+          todo.completed = !todo.completed;
+        }
+        return todo;
+      })
+    });
+  };
+
+  delTodo = (id) => {
+    this.setState({
+      todos: [...this.state.todos.filter(todo => todo.id !== id)]
+    });
+  };
+
+  addTodo = (title) => {
+    const newTodo = {
+      id: Math.random(),
```

Comment:
remove math.random() because it will add duplicated id
id:this.state.todos&&this.state.todos.length+1